src/Physics/NuElectron/XSection/NuElectronPXSec.h

```diff
@@ -57,9 +64,11 @@ class NuElectronPXSec : public XSecAlgorithmI {
   void LoadConfig (void);
 
   const XSecIntegratorI * fXSecIntegrator;
+  const ElectronVelocity * fElectronVelocity;
 
   double fSin28w; // sin^2(theta-weinberg)
   double fSin48w;
+  int N; //Number of integration samples
```

Comment:
respect the genie convetions: start the name with `f` 
`fNIntegration` is probably more descriptive than just N